easybuild/easyconfigs/s/SuiteSparse/UMFPACK_GNUMakefile-4.4.5.patch

```diff
@@ -0,0 +1,57 @@
+--- SuiteSparse/UMFPACK/Lib/GNUmakefile.org	2014-03-21 20:16:16.000000000 +0100
```

Comment:
please include a comment on top to describe what this patch file does, and include the author

(any lines above this `---` line are comments)